tools/bazel.rc

```diff
@@ -36,8 +36,9 @@ test --test_output=errors
 build --experimental_ui
 test --experimental_ui
 
-# Output to a more coventional location
-build --symlink_prefix=dist/bazel/
+# Configures script to do version stamping.
```

Comment:
do you mean to remove symlink_prefix?

Reply:
Yeah, I meant to remove it a while ago and lost the change